src/main/java/com/myhome/services/springdatajpa/CommunitySDJpaService.java

```diff
@@ -91,7 +90,7 @@ public CommunitySDJpaService(
   @Override
   public Set<String> addHousesToCommunity(String communityId, Set<CommunityHouse> houses) {
     if (!getCommunityDetailsById(communityId).isPresent()) return new HashSet<>();
-      houses.forEach(communityHouse -> communityHouse.setHouseId(generateUniqueId()));
+    houses.forEach(communityHouse -> communityHouse.setHouseId(generateUniqueId()));
```

Comment:
That operation can be done in line 96 to not loop over collection twice

Reply:
Done